packages/scene-graph-three/runtime/src/runtime_helper/loaders/FbxLoader.ts

```diff
@@ -1,8 +1,7 @@
-import * as THREE from 'three';
-import ThreeLoader from 'runtime/three/interface/ThreeLoader';
-import * as Zlib from 'runtime/three/Zlib';
-import NURBSCurve from 'runtime/three/NURBSCurve';
-import TgaLoader from 'runtime/three/loaders/TgaLoader';
+import ThreeLoader from '../interface/ThreeLoader';
```

Comment:
To use THREE, refer ThreeLoader's threeRef static property instead of using imported three module, this is because to share the same three object among scripts.